magento/pwa-studio
# Peregrine implementation/selectors be gone

## Description

Removing selector code from our repo.

`cart` selector is merged with state.
`checkout` selector is now part of the action which used it directly.
`user` selector wasn't used (part of `MyAccountMenuPage` which was replaced by `MyAccount`)
`purchaseHistory` selector wasn't used (part of `PurchaseHistory` and that is going bye bye)

## Related Issue

Closes #1632 

## Verification Steps

Make sure I deleted everything properly. 
Make sure cart and checkout still function.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
